assets/js/src/form_editor/template_selection.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import apiFetch from '@wordpress/api-fetch';
+import { GlobalContext, useGlobalContextValue } from 'context/index.jsx';
+import Notices from 'notices/notices.jsx';
+import Selection from './templates/selection';
+
+const App = () => (
+  <GlobalContext.Provider value={useGlobalContextValue(window)}>
+    <>
+      <Notices />
+      <Selection />
+    </>
+  </GlobalContext.Provider>
+);
+
+window.addEventListener('DOMContentLoaded', () => {
```

Comment:
What if DOM is loaded before you register this event? Can it happen? (jQuery's ready function is very complex to catch such scenarios)